### PR TITLE
Telehealth - fix thrown console errors (#6443)

### LINKED ISSUE #6442: Telehealth module javascript console errors.
When the video bridge shuts down it throws a javascript console error.  Sometimes when the patient waiting room shuts down, the intervals don't get cleared off quickly enough and it also throws javascript console errors on shut down.  Nothing is impacted from the UX side, but we want to make sure we clean up these errors.

### PR DESCRIPTION
**Pull request by adunsulag**
Fixes #6442

Updated the cvb library which no longer throws exceptions when the video bridge shuts down.  Fixed a couple of places where console errors were being thrown outside of the cvb library.

Bumped the script version to make sure people get the changes.

### CODE REVIEW

File: version.php [hunk 1]
```diff
@@ -47,7 +47,7 @@
     $v_js_includes = md5(microtime());
 } else {
     // Change this number when bumping
-    $v_js_includes = 72;
+    $v_js_includes = 73;
```
Comment (bradymiller) [inline version.php:50]: I'm gonna do a quick bump of this counter in master so it is different from this branch

Comment (bradymiller) [reply version.php:50]: done :)

Comment (adunsulag) [reply version.php:50]: @bradymiller should I hold off from merging? This commit was already merged in master.

**Review by bradymiller — APPROVED** [review]

File: version.php (hunk 1 above)
Comment (bradymiller) [reply version.php:50]: bring it in :)

**Pull request merged**